packages/lodestar-cli/src/util/enr.ts

```diff
@@ -0,0 +1,46 @@
+import {ENR} from "@chainsafe/discv5";
+import Multiaddr from "multiaddr";
+
+import {IBeaconNodeOptions} from "../options";
+import {IENRArgs} from "../cmds/beacon/enrOptions";
+
+export function updateENR(enr: ENR, args: IENRArgs & IBeaconNodeOptions): void {
+  if (args.network.multiaddrs.length) {
+    try {
+      const tcpOpts = Multiaddr(args.network.multiaddrs[0]).toOptions();
+      if (tcpOpts.transport === "tcp") {
+        enr.tcp = tcpOpts.port;
+      }
+    } catch (e) {
+      throw new Error("Invalid tcp multiaddr");
```

Comment:
For sanity check append the error message (at least) after the nice error message. So many things can happen in `Multiaddr`